inference-engine/src/mkldnn_plugin/nodes/mkldnn_ctc_greedy_decoder_node.cpp

```diff
@@ -42,8 +38,8 @@ MKLDNNCTCGreedyDecoderNode::MKLDNNCTCGreedyDecoderNode(const std::shared_ptr<ngr
     if (getOriginalOutputsNumber() != 1)
         IE_THROW() << errorPrefix << "has invalid number of outputs edges: " << getOriginalOutputsNumber();
 
-    if (op->get_input_shape(DATA_INDEX)[0] != op->get_input_shape(SEQUENCE_LENGTH_INDEX)[0] &&
-        op->get_input_shape(DATA_INDEX)[1] != op->get_input_shape(SEQUENCE_LENGTH_INDEX)[1])
+    if (!op->get_input_partial_shape(DATA_INDEX)[0].compatible(op->get_input_partial_shape(SEQUENCE_LENGTH_INDEX)[0]) ||
```

Comment:
use `getInputShapeAtPort/getOutputShapeAtPort` instead

`const auto& dataDims = getInputShapeAtPort(DATA_INDEX).getDims();`
`const auto& seqDims = getInputShapeAtPort(SEQUENCE_LENGTH_INDEX).getDims();`
and check
`dimsEqualWeak(dataDims[0], seqDims[0])` etc

Reply:
fix in 77986d46e02a2d3d8a43f7f35c8b76f777d0a6a4